src/pages/Home.jsx

```diff
@@ -1,91 +1,141 @@
+    limit: PAGE_LIMIT
+  }
+
+  if (selectedTag) {
+    params = Object.assign(params, { tag: selectedTag })
+  }
+  if (selectedTag) {
+    Object.assign(params, {})
+  }
+  const articlesQuery = useQuery(`get-articles-list-${offset}-${selectedTag}`, () => axios.get('https://conduit.productionready.io/api/articles', {
+    params
+  }), {
+    placeholderData: {
+      data: {
+        articles: [],
+        articlesCount: 0
+      }
+    }
+  })
+
+  const tagsQuery = useQuery('get-tags-list', () => axios.get('https://conduit.productionready.io/api/tags'), {
+    placeholderData: {
+      data: {
+        tags: []
+      }
+    }
+  })
+  const pages = articlesQuery?.data?.data?.articlesCount / PAGE_LIMIT
+  const articles = articlesQuery?.data?.data?.articles
+  const tags = tagsQuery?.data?.data?.tags
+
   return (
-    <div className="home-page">
-      <div className="banner">
-        <div className="container">
-          <h1 className="logo-font">conduit</h1>
+    <div className='home-page'>
+      <div className='banner'>
+        <div className='container'>
+          <h1 className='logo-font'>conduit</h1>
           <p>A place to share your knowledge.</p>
         </div>
       </div>
-      <div className="container page">
-        <div className="row">
-          <div className="col-md-9">
-            <div className="feed-toggle">
-              <ul className="nav nav-pills outline-active">
-                <li className="nav-item">
+      <div className='container page'>
+        <div className='row'>
+          <div className='col-md-9'>
+            <div className='feed-toggle'>
+              <ul className='nav nav-pills outline-active'>
+                <li className='nav-item'>
                   <button
-                    type="button"
-                    className="nav-link" // Add active if not feed and not tag
+                    onClick={()=>{setSelectedTag(null)}}
+                    type='button'
+                    className={classNames('nav-link',{'active':!selectedTag})} // Add active if not feed and not tag
                   >
                     Global Feed
                   </button>
                 </li>
-                <li className="nav-item">
-                  <a className="nav-link active"># {faker.lorem.word()}</a>
+                <li className='nav-item'>
+                  {selectedTag && <a className='nav-link active'># {selectedTag}</a>}
                 </li>
               </ul>
             </div>
-            <div className="article-preview">
-              <div className="article-meta">
-                <a>
-                  <img src={faker.image.avatar()} />
-                </a>
-                <div className="info">
-                  <a className="author">{faker.internet.userName()}</a>
-                  <span className="date">{new Date(faker.date.past()).toDateString()}</span>
+            {articles.map((article, idx) => {
+              return <React.Fragment key={idx}>
```

Comment:
Maybe I'm missing something but I think you don't need a Fragment here because div.article-preview wraps the whole thing.

Reply:
I do tend to add Fragment "just in case"  for when I'll need to add something later , is this a problem? bad practice?